plugins/destination/test/client/schema.json

```diff
@@ -0,0 +1,17 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://github.com/cloudquery/cloudquery/plugins/destination/test/client/spec",
+  "$ref": "#/$defs/Spec",
+  "$defs": {
+    "Spec": {
+      "properties": {
+        "error_on_write": {
+          "type": "boolean",
+          "description": "If true, will return an error on write rather than consume from the channel"
```

Comment:
you can add default here via annotating the struct:
```go
type Spec struct {
	// If true, will return an error on write rather than consume from the channel
	ErrorOnWrite bool `json:"error_on_write,omitempty" jsonschema:"default=false"`
}
```